src/components/Widgets/Mvg.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+<div class="mvg-wrapper" v-if="departures">
+  <template
+    v-for="departure in departures"
+  >
+    <div class="departure" v-bind:key="departure.key" v-tooltip="mvgTooltipDeparture(departure)">
+      <span :class="{live: departure.live}">
+        {{ departure.realtimeDepartureTime | formatDepartureTime }}
+      </span>
+    </div>
+    <div class='line'
+      v-bind:key="departure.key + 'line'"
+      >
+      <div
+        class="transport"
+        :class="['type-' + departure.transportType,
+        'line-' + departure.label,
+        ]"
+      >{{ departure.label }}</div>
+      <div
+      class='destination'
+      v-tooltip="mvgTooltipDestination(departure)"
+      :class="{cancelled: departure.cancelled}">{{ departure.destination }}</div>
+      <span class="delay"
+        :class="{'has-delay': departure.realtimeDepartureTime > departure.plannedDepartureTime}"
+      >{{ Math.max(0, 
+          (departure.realtimeDepartureTime - departure.plannedDepartureTime)/60000) }}</span>
+      <span class="occupancy"
+      :class="'occupancy-' + departure.occupancy"
+        v-if="departure.occupancy != 'UNKNOWN'"
+        v-tooltip="departure.occupancy"
+      >■</span>
+    </div>
+  </template>
+</div>
+</template>
+
+<script>
+import WidgetMixin from '@/mixins/WidgetMixin';
+import { widgetApiEndpoints } from '@/utils/defaults';
+import { timestampToTime } from '@/utils/MiscHelpers';
+
+export default {
+  mixins: [WidgetMixin],
+  components: {},
+  data() {
+    return {
+      departures: null,
+      locationSearch: null,
+    };
+  },
+  created() {
+    if (!this.isLocationId) {
+      this.makeRequest(this.endpointLocation).then(
+        (response) => {
+          const stations = response.filter((r) => r.type === 'STATION');
+          if (stations.length > 0) {
+            this.location = stations[0].globalId;
+            this.fetchData();
+          } else {
+            this.error('Cannot find station for specified string');
+          }
+        },
+      );
+    } else {
+      this.location = this.options.location;
+    }
+  },
+  filters: {
+    formatDepartureTime(timestamp) {
+      const msDifference = new Date(timestamp).getTime() - new Date().getTime();
+      const diff = Math.max(0, Math.round(msDifference / 60000));
+      return diff;
+    },
+    limitLength(str) {
+      return str;
```

Comment:
Doesn't look like this is being used, same in [`MvgConnection.vue`](https://github.com/Lissy93/dashy/blob/75b06f8af80b14f44f40b92b18df994729dd53d8/src/components/Widgets/MvgConnection.vue#L93-L95)